### PR TITLE
feat(scripts): allow reusing proxy contract in deployment script

### PR DESCRIPTION
**Pull request by milapsheth**

### CODE REVIEW

**Review by sammy1991106 — DISMISSED** [review]

File: scripts/deploy-gateway-v4.3.x.js [hunk 1]
```diff
@@ -70,6 +77,7 @@ const paramsProxy = arrayify(defaultAbiCoder.encode(['address[]', 'uint8', 'byte
     const authFactory = await getContractFactory('AxelarAuthWeighted', wallet);
     const tokenDeployerFactory = await getContractFactory('TokenDeployer', wallet);
     const gatewayProxyFactory = await getContractFactory('AxelarGatewayProxy', wallet);
+    const AxelarGateway = require('./build/AxelarGateway.json');
```
Comment (sammy1991106) [inline scripts/deploy-gateway-v4.3.x.js:80]:
```suggestion
    const IAxelarGateway = require('./build/IAxelarGateway.json');
```

Comment (sammy1991106) [reply scripts/deploy-gateway-v4.3.x.js:80]: should pull the interface instead

Comment (re1ro) [reply scripts/deploy-gateway-v4.3.x.js]: I've fixed it by using `gatewayFactory`

**Review by re1ro — DISMISSED** [review]

**Comment by re1ro** [comment]
@milapsheth is this PR still needed or can be closed?

**Comment by milapsheth** [comment]
closed in favour of #170

**Pull request closed without merging**